tests/tests/atmaero_control_p8

```diff
@@ -91,6 +91,7 @@ export IOPT_STC=3
 # P8
 export IOPT_SFC=3
 export IOPT_TRS=2
+export IOPT_DIAG=2
```

Comment:
I think IOPT_DIAG and FNSOCC are defined in default_vars.sh, do we still need to define them here?

Reply:
Yes. Control P8 doesn't use all default settings.

Reply:
@HelinWei-NOAA I think I will go ahead to resolve this conversation.